src/cmd.c

```diff
@@ -1802,6 +1825,7 @@ static const atci_command_t cmds[] = {
 #endif
     {"$CERT",        NULL,    set_cert,         get_cert,         NULL, "Enable or disable LoRaWAN certification port"},
     {"$SESSION",     NULL,    NULL,             get_session,      NULL, "Get network session information"},
+    {"$CW",          cw,      NULL,             NULL,             NULL, "Continuous carrier"},
```

Comment:
Language nit pick: "Start continuous carrier wave transmission".

Reply:
Was trying to save a few bytes ;) Thanks for copy-paste solution.